<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TS_RegressionTest" time="4.508" tests="3" failures="0" errors="3">
   <testsuite name="TS_RegressionTest" tests="3" failures="0" errors="3" time="4.508" timestamp="2020-07-14 22:36:10" hostname="ntn64 - DESKTOP-43BOAOC" id="Test Suites/TS_RegressionTest">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\ntn64\\Katalon Studio\\HW3\\Reports\\20200714_223600\\TS_RegressionTest\\20200714_223600"/>
         <property name="logFiles" value="C:\\Users\\ntn64\\Katalon Studio\\HW3\\Reports\\20200714_223600\\TS_RegressionTest\\20200714_223600\\execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="ntn64 - DESKTOP-43BOAOC"/>
         <property name="hostAddress" value="192.168.1.4"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="7.4.0.1"/>
      </properties>
      <testcase name="Test Cases/03 - 003 Chatbox - ki\u1EC3m tra khi nh\u1EADp b\u1ECF tr\u1ED1ng" time="1.775" classname="Test Cases/03 - 003 Chatbox - ki\u1EC3m tra khi nh\u1EADp b\u1ECF tr\u1ED1ng" status="ERROR">
         <error type="ERROR" message="Test Cases/03 - 003 Chatbox - kiểm tra khi nhập bỏ trống FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 003 Chatbox - kiểm tra khi nhập bỏ trống.run(03 - 003 Chatbox - kiểm tra khi nhập bỏ trống:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)
"/>
         <system-out><![CDATA[2020-07-14 22:36:10 - [TEST_CASE][ERROR] - Test Cases/03 - 003 Chatbox - ki\u1EC3m tra khi nh\u1EADp b\u1ECF tr\u1ED1ng: Test Cases/03 - 003 Chatbox - kiểm tra khi nhập bỏ trống FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 003 Chatbox - kiểm tra khi nhập bỏ trống.run(03 - 003 Chatbox - kiểm tra khi nhập bỏ trống:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)


2020-07-14 22:36:12 - [TEST_STEP][PASSED] - callTestCase(findTestCase("1612419/03 - 006 Login success (reusable)"), ["url":"http://localhost:8065/login", "username":"ntn641998", "password":"Hcmushcmus@123"], STOP_ON_FAILURE): null]]></system-out>
         <system-err><![CDATA[2020-07-14 22:36:10 - [TEST_CASE][ERROR] - Test Cases/03 - 003 Chatbox - ki\u1EC3m tra khi nh\u1EADp b\u1ECF tr\u1ED1ng: Test Cases/03 - 003 Chatbox - kiểm tra khi nhập bỏ trống FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 003 Chatbox - kiểm tra khi nhập bỏ trống.run(03 - 003 Chatbox - kiểm tra khi nhập bỏ trống:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)]]></system-err>
      </testcase>
      <testcase name="Test Cases/03 - 002 Chatbox - ki\u1EC3m tra c\u00F3 hi\u1EC3n th\u1ECB tin nh\u1EAFn m\u1EDBi nh\u1EADp" time="1.193" classname="Test Cases/03 - 002 Chatbox - ki\u1EC3m tra c\u00F3 hi\u1EC3n th\u1ECB tin nh\u1EAFn m\u1EDBi nh\u1EADp" status="ERROR">
         <error type="ERROR" message="Test Cases/03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập.run(03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)
"/>
         <system-out><![CDATA[2020-07-14 22:36:12 - [TEST_CASE][ERROR] - Test Cases/03 - 002 Chatbox - ki\u1EC3m tra c\u00F3 hi\u1EC3n th\u1ECB tin nh\u1EAFn m\u1EDBi nh\u1EADp: Test Cases/03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập.run(03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)


2020-07-14 22:36:13 - [TEST_STEP][PASSED] - callTestCase(findTestCase("1612419/03 - 006 Login success (reusable)"), ["url":"http://localhost:8065/login", "username":"ntn641998", "password":"Hcmushcmus@123"], STOP_ON_FAILURE): null]]></system-out>
         <system-err><![CDATA[2020-07-14 22:36:12 - [TEST_CASE][ERROR] - Test Cases/03 - 002 Chatbox - ki\u1EC3m tra c\u00F3 hi\u1EC3n th\u1ECB tin nh\u1EAFn m\u1EDBi nh\u1EADp: Test Cases/03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập.run(03 - 002 Chatbox - kiểm tra có hiển thị tin nhắn mới nhập:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)]]></system-err>
      </testcase>
      <testcase name="Test Cases/03 - 001 Chatbox - ki\u1EC3m tra c\u00F3 clear \u00F4 nh\u1EADp sau khi enter" time="0.864" classname="Test Cases/03 - 001 Chatbox - ki\u1EC3m tra c\u00F3 clear \u00F4 nh\u1EADp sau khi enter" status="ERROR">
         <error type="ERROR" message="Test Cases/03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter.run(03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)
"/>
         <system-out><![CDATA[2020-07-14 22:36:13 - [TEST_CASE][ERROR] - Test Cases/03 - 001 Chatbox - ki\u1EC3m tra c\u00F3 clear \u00F4 nh\u1EADp sau khi enter: Test Cases/03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter.run(03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)


2020-07-14 22:36:14 - [TEST_STEP][PASSED] - callTestCase(findTestCase("1612419/03 - 006 Login success (reusable)"), ["url":"http://localhost:8065/login", "username":"ntn641998", "password":"Hcmushcmus@123"], STOP_ON_FAILURE): null]]></system-out>
         <system-err><![CDATA[2020-07-14 22:36:13 - [TEST_CASE][ERROR] - Test Cases/03 - 001 Chatbox - ki\u1EC3m tra c\u00F3 clear \u00F4 nh\u1EADp sau khi enter: Test Cases/03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter.run(03 - 001 Chatbox - kiểm tra có clear ô nhập sau khi enter:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)]]></system-err>
      </testcase>
      <system-out><![CDATA[2020-07-14 22:36:10 - [TEST_SUITE][ERROR] - TS_RegressionTest: Test Cases/03 - 003 Chatbox - kiểm tra khi nhập bỏ trống FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 003 Chatbox - kiểm tra khi nhập bỏ trống.run(03 - 003 Chatbox - kiểm tra khi nhập bỏ trống:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)]]></system-out>
      <system-err><![CDATA[2020-07-14 22:36:10 - [TEST_SUITE][ERROR] - TS_RegressionTest: Test Cases/03 - 003 Chatbox - kiểm tra khi nhập bỏ trống FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test case 'Test Cases/1612419/03 - 006 Login success (reusable)'
	at com.kms.katalon.core.testcase.TestCaseFactory.findTestCase(TestCaseFactory.java:72)
	at com.kms.katalon.core.testcase.TestCaseFactory$findTestCase.callStatic(Unknown Source)
	at 03 - 003 Chatbox - kiểm tra khi nhập bỏ trống.run(03 - 003 Chatbox - kiểm tra khi nhập bỏ trống:23)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:337)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:307)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:299)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:233)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:169)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:142)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:91)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:157)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1594740960424.run(TempTestSuite1594740960424.groovy:37)]]></system-err>
   </testsuite>
</testsuites>
